manifests/charts/base/templates/zzz_profile.yaml

```diff
@@ -0,0 +1,19 @@
+{{/*
+Complex logic ahead...
```

Comment:
Interesting approach! Should we briefly include the merge logic in the README as well?